views/layout.pug

```diff
@@ -0,0 +1,20 @@
+doctype html
+html
+  head
+    meta( charset='utf-8' )
+    meta( http-equiv='X-UA-Compatible', content='IE=edge' )
+    meta(name='viewport', content='width=device-width, initial-scale=1')
+
+    title= title
+
+    link(rel='stylesheet', href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css', integrity='sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ', crossorigin='anonymous')
```

Comment:
Normally I'd say "don't add libraries to a PR unless the code actually uses them". Because anticipating future needs usually translates to accumulating stale code.

But if you are going to make use of jQuery & Bootstrap right away, then go ahead and keep these here.